pkg/k8shandler/visualization.go

```diff
@@ -23,8 +24,12 @@ import (
 )
 
 const (
-	kibanaServiceAccountName     = "kibana"
-	kibanaOAuthRedirectReference = "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"kibana\"}}"
+	annotationOauthSecretUpdatedAt = "logging.openshift.io/oauthSecretUpdatedAt"
```

Comment:
what are we going to use this for?
does this have overlap with https://github.com/openshift/cluster-logging-operator/pull/351 ?

Reply:
Why is this in this change?  We added "oauth" bits in a separate PR  so the oauthclient would be dynamically generated.  I wold not expect any "oauthclient" parts to change in order to satisfy this bz

Reply:
This looks like a mistake caused by resolving a merge conflict. Good catch! PR should be updated now.